shared/rakefiles/xk_util.rake

```diff
@@ -51,3 +51,22 @@ task :rotate_secrets_key, [:encryption_key] => [:configure_serviceaccount] do |t
   Secrets.set_secrets(@secrets, rotate_secrets = true)
   Secrets.disable_non_primary_key_versions(args[:encryption_key], new_version_id)
 end
+
+# This task destroy all keys except current one for projectowner's SA.
+# All operations will be ignored in case local credentials not present or SA usage not configured.
```

Comment:
How about this wording: "This task does nothing unless local credentials are present or a Serviceaccount is configured."

Reply:
Changed to `It does nothing in case local SA credentials not present.`